language-server/modules/langserver-core/src/test/resources/completion/annotation/source/annotationInSameModule4.bal

```diff
@@ -39,6 +39,11 @@ type Listener object {
     public function __start() returns error? {
     }
 
-    public function __stop() returns error? {
+    public function __gracefulStop() returns error? {
+        return ();
```

Comment:
We don't need this `return()` similar to `__start()`. Same for `__immediateStop` too.